Source/WebCore/rendering/RenderGrid.cpp

```diff
@@ -698,16 +698,45 @@ static GridArea insertIntoGrid(Grid& grid, RenderBox& child, const GridArea& are
     return clamped;
 }
 
-static inline bool isMasonryRows(const RenderStyle& style)
+bool RenderGrid::isMasonryRows() const
```

Comment:
Nit: This naming is awkward. Really should be `isMasonryRow` or `areMasonryRows` or something. But not worth changing this patch for.